docs/prover/checking/coverage-info.md

```diff
@@ -0,0 +1,107 @@
+# Coverage Info
+
+The {ref}`--coverage_info` flag enables automatic computation of coverage
+information for a verification run. In particular, using this flag can help you
+answer questions such as:
+
+* *Are all solidity functions from the input involved in proving my rules?*
+* *Are all solidity commands from the input involved in proving my rules?*
+* *Supposing an `assert` in my rule is not reachable, what is the reason for the
+  unreachability?*
+* *Do I really need all hooks that are defined in my `.spec` file(s)?*
+* *Do I really need all `require` statements in my rule?*
+* *Do I really need to initialize a CVL variable in my rule?*
+* *Do I really need all of the preserved blocks in my CVL `invariant`?*
+
+To answer the above questions, the Certora Prover identifies a minimal subset of
+the commands in the input `.sol` and `.spec` files that are relevant for proving
+the CVL properties. If some of the input `.sol` commands are not relevant for
+deriving the proof, it might indicate that the specification does not cover all
+behavior implemented in the smart contract. If some of the input `.spec`
+commands are irrelevant (typically unnecessary `require` statements or variable
+initializations), it indicates that the CVL rules/invariants can be made
+stronger. 
+
+You can access the coverage visualization via `Job Info -> Coverage Info page`
+buttons:
+
+![Coverage Info Button](coverage-info-button.png)
+
+
+## Examples
+
+### Tautology example
+Consider the following CVL rule called `tautology`: 
+
+```cvl
+rule tautology(uint256 fundId, method f) { 
```

Comment:
This is a complex enough example that I'd consider turning on line numbers and referring to them in the text below (e.g. "we required `other != manager` on line 5").

To do so, I think the syntax is
````
```{code-block} cvl
:linenos:

rule tautology(...) {
 ...
}
```
````